cli/dcoscli/task/main.py

```diff
@@ -148,6 +150,17 @@ def _log(follow, completed, lines, task, file_):
                 raise DCOSException(msg)
         raise DCOSException('No matching tasks. Exiting.')
 
+    if file_ in ('stdout', 'stderr') and log.dcos_log_enabled():
+
+        try:
```

Comment:
same here, if dcos_log throws an exception, we should be able to query the files API. Let me know about this one as well